app/Phragile/Burndown.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Phragile;
+
+class Burndown {
```

Comment:
The name will probably be clearer if you make it a bit more noun like. Such as BurndownChart, if that is what this class actually is supposed to represent.